spec/workflow/payload_template_spec.rb

```diff
@@ -20,6 +20,16 @@
       end
     end
 
+    context "with intrinsic functions" do
+      let(:payload) { {"uuid.$" => "States.UUID()"} }
+      let(:context) { {} }
+      let(:inputs)  { {} }
+
+      it "returns the value from the inputs" do
+        expect(subject.value(context, inputs)["uuid"]).to match(/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/)
```

Comment:
Super-minor, but I have a slightly smaller regex that I used in the uuid specs.  For consistency let's reuse that one (or we can move into a spec/support thing or perhaps use rspec-uuid gem that includes a matcher).  Not necessary for this PR though.

Reply:
:+1: done